sources/packages/web/src/composables/institution/useInstitutionState.ts

```diff
@@ -1,9 +1,9 @@
 import { InstitutionStateForStore, LocationStateForStore } from "@/types";
 import { computed } from "vue";
-import { useStore } from "vuex";
+import { Store, useStore } from "vuex";
 
-export function useInstitutionState() {
-  const store = useStore();
+export function useInstitutionState(rootStore?: Store<any>) {
```

Comment:
Can we replace `any` with `unknown`?

Reply:
I did that and saw the following error. Felt that creating an interface and typing that with it could be done later.

![image](https://user-images.githubusercontent.com/54600590/233219579-a97ce38a-6251-4860-84ca-c66efeee1ff6.png)